qemu/tests/cfg/boot_cpu_model.cfg

```diff
@@ -49,9 +49,13 @@
                 - remove_flag:
                     cpu_model_GenuineIntel = "SandyBridge"
                     cpu_model_AuthenticAMD = "Opteron_G3"
-                    cpu_model_flags = ",-sse4a,-xsave,-aes,-sse4.2"
+                    cpu_model_flags = ",-sse4a,-xsave,-aes,-sse4.2,+ht,+sep"
+                    cpu_model_flags_AuthenticAMD = ",-monitor,+vme,+lahf_lm,+cmp_legacy"
                 - cpuid_7_0_ebx_feature:
                     cpu_model_flags = ",+fsgsbase"
                     cpu_model = "SandyBridge"
                 - cpu_host:
                     cpu_model = "host"
+                    cpu_model_flags = ",+ht,+fma,+bmi1"
```

Comment:
Did you test in other host, what's the result if test on Intel host or host < Optern_G5